examples/DynamicCantileverCase/dynamic_cantilever_phase_space.py

```diff
@@ -0,0 +1,84 @@
+import matplotlib.pyplot as plt
+from dynamic_cantilever import simulate_dynamic_cantilever_with
+
+if __name__ == "__main__":
+    import multiprocessing as mp
+
+    PLOT_FIGURE = True
+    SAVE_FIGURE = False
+
+    # density = 500, 1000, 2000, 3000, 4000, 5000 kg/m3
+    densities = [500]
+    densities.extend(range(1000, 6000, 1000))
+
+    with mp.Pool(mp.cpu_count()) as pool:
+        results = pool.map(simulate_dynamic_cantilever_with, densities)
+
+    thry_frequency = []
+    sim_frequency = []
+    thry_amplitude = []
+    sim_amplitude = []
+
+    for result in results:
+        thry_frequency.append(result["theoretical_frequency"])
+        sim_frequency.append(result["simulated_frequency"])
+        thry_amplitude.append(result["theoretical_amplitude"])
+        sim_amplitude.append(result["simulated_amplitude"])
+
```

Comment:
Lets put a command here saying rest of the script is about plotting. Something like `Plot the density vs frequency`

Reply:
[0574bb3](https://github.com/GazzolaLab/PyElastica/pull/173/commits/0574bb3a5bead204311a6ae3570c4e3a08cd980c)